sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -48,11 +48,27 @@ class SettingSelector:
     :keyword label_filter: A filter to select configuration settings based on their labels. Default is value is
      EMPTY_LABEL i.e. (No Label) as seen in the portal.
     :type label_filter: Optional[str]
+    :keyword tag_filters: A filter to select configuration settings based on their tags. This is a list of strings
+     that will be used to match tags on the configuration settings. Reserved characters (\\*, \\, ,) must be escaped
+     with backslash if they are part of the value.
```

Comment:
It would be helpful to add an example of filtering tags with null value and empty value

Reply:
How is the new version.